apps/condo/domains/property/components/AddressSuggestionsSearchInput.tsx

```diff
@@ -73,11 +71,11 @@ export const AddressSuggestionsSearchInput: React.FC<AddressSearchInputProps> =
     )
 
     const renderOption = useCallback(
-        (dataItem: TSelectItem, searchValue) => {
+        (dataItem: TSelectedAddressSuggestion, searchValue) => {
             return (
                 <Select.Option
                     style={{ direction: 'rtl', textAlign: 'left', color: grey[6] }}
-                    key={dataItem.value}
+                    key={JSON.stringify(dataItem)}
```

Comment:
strange key

Reply:
Earlier the key was almost the same stringified object. It's ok.